docs/microk8s/sc4snmp-installation.md

```diff
@@ -67,6 +67,43 @@ rules:
 Configuration above can be found in the `examples` directory in SC4SNMP [GitHub repository](https://github.com/splunk/splunk-connect-for-snmp).
 Next run `yamllint -c <path to custom-config.yamllint> <path to values.yaml>` command. Warnings can be ignored.
 
+
+### Protect Mongo and Redis by password
+
+Create secrets for Mongo and Redis:
+
+```
+kubectl create secret generic redis-auth-secret \
```

Comment:
Wouldn't be better to create some default secret so we don't have problems during migration? and note that they can change it and how? I know it can be a problem for prodsec but still default secret > no secret 😅

Reply:
I think additional secrets will confuse people more, do you agree?